examples/3d_kymograph_.py

```diff
@@ -7,15 +7,14 @@
 
 .. tags:: experimental
 """
-from typing import Dict, List, Tuple
 import numpy as np
 import napari
 from tqdm import tqdm
 from itertools import product
 
 try:
     from omero.gateway import BlitzGateway
-except:
+except ImportError:
```

Comment:
```suggestion
except ModuleNotFoundError:
```

`ImportError` could mean any other issue importing, and can hide problems.